packages/devtools_app/test/memory/diff/widgets/class_filter_test.dart

```diff
@@ -39,63 +33,81 @@ final _tests = [
   _FilterTest(isDiff: true),
 ];
 
+final _customFilter = ClassFilter(
+  filterType: ClassFilterType.only,
+  except: '',
+  only: '',
+);
+
 void main() {
-  late DiffSnapshotScene scene;
+  final scene = DiffSnapshotScene();
+  // The setup should happen one time, so that there is only one instance of
+  // DiffPaneController accross tests.
+  // Otherwize tests may exchange controllers.
```

Comment:
can you add a TODO to debug this further? This is a bug with how we are managing state that we should fix